tests/integration-tests/tests/common/mpi_common.py

```diff
@@ -51,8 +51,10 @@ def _test_mpi(
         scheduler_commands.assert_job_succeeded(job_id)
 
     mpi_out = remote_command_executor.run_remote_command("cat /shared/mpi.out").stdout
```

Comment:
_minor_ can you please add a mpi_out as example?

Reply:
done